src/models/music/Playlist.ts

```diff
@@ -53,6 +53,22 @@ export class Playlist {
     return true;
   }
 
+  /**
+   ** shuffle tracks
+   */
```

Comment:
Useless comment, document what this function actually does.

Reply:
"Randomize the order of tracks in the current playlist"